exopy/__main__.py

```diff
@@ -269,6 +273,10 @@ def main(cmd_line_args=None):
         details = format_exc()
         display_startup_error_dialog(text, content, details)
 
+    # Quit hard and early if we are headless mode
+    if args.execute:
```

Comment:
Wouldn't it be better to skip only the GUI part but keep the clean exit ?

Reply:
I think we tested that and it required more code and was slower for the exact same result so we stuck with this simple approach.